### PR TITLE
Don't run `bypass-checks` if it does not link to the checks

### LINKED ISSUE #2967: `bypass-checks` fails when cors is needed
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/654 (or any pull there)
```console
Access to fetch at 'https://codecov.io/gh/sindresorhus/eslint-plugin-unicorn/compare/b2838fdff7ecdc71008e049c9b1ad57da11aa224...2abde3bbf93d00305d0e85701c4b7be383fa9ed7' from origin 'https://github.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
```js
refined-github.js:4104 ❌ Refined GitHub → bypass-checks → TypeError: Failed to fetch 
```

### PR DESCRIPTION
**Pull request by yakov116**
@fregante I dont think this is what you had in mind

Fix #2967
<!--

Thanks for contributing! 🍄

1. LINKED ISSUES:
   Does this PR close/fix an existing issue? Write something like `Closes #10`

2. TEST URLS:
   Add some test URLs

3. SCREENSHOT:
   Add a screenshot here if your PR makes visual changes

### CODE REVIEW

**Comment by fregante** [comment]
The point of this feature is bypass the Checks page. This is achieved by fetching the Checks page and then extracting its final URL from it.

If the link doesn't point to Checks, then there's nothing to bypass.

Therefore no fetch should be made at all.

File: source/features/bypass-checks.tsx [hunk 1]
```diff
@@ -9,8 +9,10 @@ async function init(): Promise<void> {
 
 async function bypass(check: HTMLElement): Promise<void> {
 	const details = select<HTMLAnchorElement>('.status-actions', check.parentElement!)!;
```
Comment (fregante) [inline source/features/bypass-checks.tsx]:
```suggestion
	const details = select<HTMLAnchorElement>('.status-actions', check.parentElement!)!;
	if (details.origin !== location.origin) {
		return;
	}

```

Comment (yakov116) [reply source/features/bypass-checks.tsx]: ohh that what you meant

Comment (yakov116) [reply source/features/bypass-checks.tsx]: But travis would pass?

**Comment by yakov116** [comment]
@fregante LOL i just understood what this feature is doing :) (I'm slow... sometimes)

**Pull request closed without merging**